cmd/scanners/aws/scanner.go

```diff
@@ -106,45 +110,55 @@ func startStockers() error {
 		if err != nil {
 			return err
 		}
-		// TODO handle error properly
 		inven.AddAccount(stockerInstance.GetResults())
 	}
 	return nil
 }
 
 func main() {
-	defer logger.Sync()
+	// Ignore Logger sync error
+	defer func() { _ = logger.Sync() }()
```

Comment:
What's the difference compared previous defer?

Reply:
Errors in the defer also need to be handled
I just explicitly indicated that we are skipping it